### PR TITLE
delete custom target yamls

### PR DESCRIPTION
**Pull request by OdeclasV**
Forgot to open this PR yesterday before signing off. The comment deadlines for these guys have passed.

Feel free to approve and merge

### CODE REVIEW

**Review by vramoscoreas — APPROVED** [review]

**Comment by vramoscoreas** [comment]
Will wait for @tirish  to give his approval before merging.

**Review by tirish — APPROVED** [review]

**Pull request merged**
